eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -64,7 +79,7 @@ steps:
         arguments: >
           -Organization "azure-sdk"
           -Project "internal"
-          -SourceBranch "${{ parameters.UpstreamBranchName }}"
+          -SourceBranch "${{ parameters.UpstreamBranchName }}-ForTestPipeline"
```

Comment:
If we start using these branches we should add some code at the end of the pipeline that removes these branches.